src/components/ConfigProvider/Theming/Theming.types.ts

```diff
@@ -35,6 +42,7 @@ export interface OcBaseTheme {
     errorColor?: Value;
     tabsTheme?: TabsTheme;
     navbarTheme?: NavbarTheme;
+    varTheme?: VarTheme;
```

Comment:
Might also consider adding buttonTheme and tableTheme, etc. That way we have the potential of global varTheme and scoped component themes. Also ensures previous work scales.

Reply:
per office hour discussion, i'm not sure I want to carry this existing pattern forward, implementation of it has more touchpoints/complications, including in Eightfold configurations.